README.md

```diff
@@ -0,0 +1,246 @@
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> List of technologies used for this project
+
+<details>
+  <summary>BackEnd</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org">Ruby on Rails</a></li>
+    <li><a href="https://stimulus.hotwired.dev/">Stimulus</a></li>
+    <li><a href="https://rspec.info/">Rspec</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Front-end/View</summary>
+  <ul>
+    <li><a href="https://react.dev/">React</a></li>
+    <li><a href="https://axios-http.com/es/docs/intro">Axios</a></li>
+    <li><a href="https://tailwindcss.com/docs/guides/ruby-on-rails">Tailwind CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Front-end/Store management</summary>
+  <ul>
+    <li><a href="https://react-redux.js.org/">React-redux</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Create and Remove Games**
+- **Create Reservations**
+- **Full stack webpage**
+- **Can see all the list of reservation of current user**
+- **Details of every game in db**
+- **Log in and Sign up feature**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Under Construction](TBA) 💻📲
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🎫 Kanban Board <a name="kanban-board"></a>
+
+- [Link to Kanban Board.](https://github.com/users/cancelei/projects/7)
+- [Initial state of the Kanban Board.](https://github.com/cancelei/EsportGamingCenter_backend/assets/45726631/17119158-dc80-416e-a44a-8fc5b4b205c3)
+- *We are 5 members in our Team.*
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+you need to install ruby 3.2.2 using the prefer tool, example: 
+- [rvm](https://rvm.io/)
+- [rbenv](https://github.com/rbenv/rbenv)
+- [asdf-vm](https://asdf-vm.com/)
+
+- Browser (Google Chrome, Mozilla Firefox, Safari or any other browser)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd my-folder
+  git clone https://github.com/cancelei/EsportGamingCenter_frontend
+```
+
+### Install
+
+Install this project with:
+
+
+```sh
+  cd EsportGamingCenter_backend
+  bundle install
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+./bin/dev
+```
+**Note:** If there are any errors consider adding your necesary username and password in `database.yml` for setting up your local enviroment and use the following comand:
+
+```sh
+rails db:drop db:create db:migrate
+./bin/dev
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+rspec ./spec/models
+```
```

Comment:
- [ ] These are instructions for the backend project that won't work in the frontend project.  Kindly remove them to prevent confusion. 👍